### DISCUSSION TITLE
Projects (Beta): Embed task list progress into Project card

### DISCUSSION THREAD

**Discussion by MichielDeMey** [community/community#5526] [Projects and Issues]
I currently have an issue where I have a bunch of tasks using the new task list.
While we can see the task list progress clearly in the issue details, it is not reflected on the card itself.

I believe it would be a nice addition to include the task progress on the project's issue card.

Project issue card (**where we could show task list progress**):
<img width="346" alt="Screenshot 2021-09-08 at 15 28 29" src="https://user-images.githubusercontent.com/793406/132518702-ff32c1c6-b899-4f17-987a-cde4ac45ff6b.png">

Issue details (**notice 17 tasks done**):
<img width="943" alt="Screenshot 2021-09-08 at 15 28 43" src="https://user-images.githubusercontent.com/793406/132518748-1c8c6ce6-69c7-4597-a21c-5e566453d5de.png">

**Comment by ElijahLynn** [comment]
Just came here to post this exact same thing! Thanks for posting this. This normally would be a comment, but the Answers format doesn't allow for that, so oh well. :shrug:

**Reply by ddelange** [reply]
duplicate of https://github.com/github/feedback/discussions/7575 ?

**Reply by MichielDeMey** [reply]
> duplicate of https://github.com/github/feedback/discussions/7575 ?

I believe that chronologically, that's a duplicate of this discussion. 🙃
